packages/agents/sequencer/src/sequencer.ts

```diff
@@ -122,7 +124,12 @@ export const execute = async (_configOverride?: SequencerConfig) => {
     context.adapters.contracts = getContractInterfaces();
     context.adapters.relayer = await setupRelayer();
 
-    await executeAuction(transferId, requestContext);
+    if (messageType == MessageType.Auction) {
+      await executeAuction(transferId, requestContext);
+    } else if (messageType == MessageType.SlowPath) {
```

Comment:
OK I see here these are namespaced, nice!

Reply:
Nit: That would be better to update MessageType.Auction like MessageType.FastPath?

Reply:
Yes, correct this would be better!